### PR TITLE
PMM-6014 Changes for RC on OVF and AMI builds

### PR DESCRIPTION
**Pull request by puneet0191**
This PR adds repo parameter which will be used for build the OVF and AMI images

1) Adds an option to prepare for Release Candidate
2) Sets the build description to Release candidate, makes it easy to identify
3) Sends a message in pmm-qa to let QA team keep track for RC build. 

A test job was executed to check the changes [here](https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ami-temp/detail/pmm2-ami-temp/11/pipeline) 

This PR should be combined with the changes proposed on percona-images https://github.com/Percona-Lab/percona-images/pull/100

### CODE REVIEW

File: pmm/pmm2-ovf.groovy [hunk 1]
```diff
@@ -31,11 +35,38 @@ pipeline {
...
+            when {
+                expression { env.RELEASE_CANDIDATE == "yes" }
+            }
+            steps {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                    sh """
+                        packer build \
+                        -var 'pmm_client_repos=original testing' \
+                        -var 'pmm_client_repo_name=percona-testing-x86_64' \
+                        -var 'pmm2_server_repo=laboratory' \
```
Comment (denisok) [inline pmm/pmm2-ovf.groovy:48]: should be `testing` ?

File: pmm/pmm2-ami.groovy [hunk 2]
```diff
@@ -37,12 +41,40 @@ pipeline {
...
+            }
             steps {
                 withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
                     sh """
                         set -o pipefail
-                        ~/bin/packer build -only amazon-ebs -color=false packer/pmm2.json \
+                        ~/bin/packer build \
+                        -var 'pmm_client_repos=original testing' \
+                        -var 'pmm_client_repo_name=percona-testing-x86_64' \
+                        -var 'pmm2_server_repo=laboratory' \
```
Comment (denisok) [inline pmm/pmm2-ami.groovy:55]: testing ?

File: pmm/pmm2-ovf.groovy (hunk 1 above)
Comment (puneet0191) [reply pmm/pmm2-ovf.groovy:48]: yes, updating..thanks

File: pmm/pmm2-ami.groovy (hunk 2 above)
Comment (puneet0191) [reply pmm/pmm2-ami.groovy:55]: updating, thanks a lot

File: pmm/pmm2-ovf.groovy [hunk 3]
```diff
@@ -31,11 +35,38 @@ pipeline {
...
+                    sh """
+                        packer build \
+                        -var 'pmm_client_repos=original testing' \
+                        -var 'pmm_client_repo_name=percona-testing-x86_64' \
+                        -var 'pmm2_server_repo=testing' \
+                        -only virtualbox-ovf -color=false packer/pmm2.json \
+                            | tee build.log
+                    """
+                }
+                sh 'ls */*.ova | cut -d "/" -f 2 > IMAGE'
```
Comment (Sudokamikaze) [inline pmm/pmm2-ovf.groovy:53]: Better to replace by `find` instead of `ls + cut` construction

**Review by denisok — APPROVED** [review]

File: pmm/pmm2-ovf.groovy (hunk 3 above)
Comment (puneet0191) [reply pmm/pmm2-ovf.groovy:53]: I haven't really changed anything on this, just using the same way as we do for current builds.

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Review by denisok — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**